karma.conf.js

```diff
@@ -0,0 +1,106 @@
+module.exports = function(config) {
+
+  'use strict';
+  var ramdaMain = './node_modules/ramda/dist/ramda.js';
+  var sanctuaryDefMain = './node_modules/sanctuary-def/index.js';
+
+  // https://saucelabs.com/platforms
+  var customLaunchers = {
+    sl_chrome: {
+      base: 'SauceLabs',
+      browserName: 'chrome',
+      version: '51'
+    },
+    sl_firefox: {
+      base: 'SauceLabs',
+      browserName: 'firefox',
+      version: '47'
+    },
+    sl_Safari_8: {
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OS X 10.10',
+      version: '8'
+    },
+    SL_Safari_10: {
+      base: 'SauceLabs',
+      browserName: 'safari',
+      platform: 'OS X 10.11',
+      version: '10'
+    },
+    SL_iOS_Safari_8: {
+      base: 'SauceLabs',
+      browserName: 'iphone',
+      version: '8.1'
+    },
+    SL_IE_9: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      platform: 'Windows 2008',
+      version: '9'
+    },
+    sl_ie_11: {
+      base: 'SauceLabs',
+      browserName: 'internet explorer',
+      platform: 'Windows 8.1',
+      version: '11'
+    }
+  };
+
+  config.set({
+
+    browserDisconnectTimeout: 10000,
+    browserDisconnectTolerance: 2,
+    browserNoActivityTimeout: 90000,
+    captureTimeout: 120000,
+
+    client: {
+      mocha: {opts: 'test/mocha.opts'}
+    },
+
+    frameworks: [
+      'browserify',
+      'mocha'
+    ],
+
+    files: [
+      ramdaMain,
+      sanctuaryDefMain,
+      'index.js',
+      'test/**/*.js'
+    ],
+
+    preprocessors: {
+      'test/**/*.js': ['browserify']
+    },
+
+    reporters: [
+      'dots',
+      'saucelabs'
+    ],
+
+    singleRun: true,
+
+    browserify: {
+      configure: function(bundle) {
+        bundle.on('prebundle', function() {
+          bundle.require(ramdaMain, {expose: 'ramda'});
+          bundle.require(sanctuaryDefMain, {expose: 'sanctuary-def'});
```

Comment:
I've realized it's possible to avoid hard-coding these paths. What do you think of this approach?

```javascript
var depMain = function(name) {
  var pkg = require(name + '/package.json');
  var main = 'main' in pkg ? pkg.main : 'index.js';
  return './node_modules/' + name + '/' + main;
};
```

We could then replace the two lines above with the following:

```javascript
for (var name in require('./package.json').dependencies) {
  bundle.require(depMain(name), {expose: name});
}
```

We could use `depMain` to specify `files` as well.

This approach is a bit more complex, but would hopefully save us from having to change this file if we add or remove a dependency or if the layout of one of our dependencies changes.